static/scss/answers/module.scss

```diff
@@ -31,6 +31,53 @@
       @include TextButton($padding: 0);
   }
 
+  &-actions
+  {
+    display: flex;
+    align-items: flex-end;
+
+    @include bpgte(sm)
+    {
+      flex-direction: column;
+    }
+  }
+
```

Comment:
can we double check if the styling is good on mobile and small/wide screen browser page?

Reply:
yes, it looks good in all 3 cases